CMakeLists.txt

```diff
@@ -21,6 +21,8 @@ if(CUDAToolkit_FOUND AND NOT(USE_CPU))
   set(MaCh3_GPU_ENABLED TRUE)
   enable_language(CUDA)
   set(CPU_ONLY FALSE)
+  #Create a seperate target for GPU compiler options
+  add_library(MaCh3GPUCompilerOptions INTERFACE)
```

Comment:
can you just move it away from if GPU?

But make it empty for CPU only build? I wonder if this will not simplyfy logic

Reply:
yep good point! I've changed this both in the main CMakeList.txt and also manager/CMakeLists.txt and splines/CMakeLists.txt